dunstrc

```diff
@@ -228,6 +228,12 @@
     # Maximum amount of notifications kept in history
     history_length = 20
 
+    ### DND ###
```

Comment:
As @roobre noted in their comment, it's better to move this example to the rules section. You could even set some basic defaults like 30 for normal notifications and 60 for urgent notifications